aspnetcore/fundamentals/middleware.md

```diff
@@ -125,7 +129,7 @@ The following table shows the requests and responses from `http://localhost:1234
 | localhost:1234 | Hello from non-Map delegate.  |
 | localhost:1234/?branch=master | Branch used = master|
 
-You can nest Maps:
+`Map` can also match multiple segments at once, for example:
```

Comment:
Slightly too literal... This sample is for nesting, I was recommending a variation of the sample showing Map matching multiple segments.